src/components/cards/cards.css

```diff
@@ -0,0 +1,177 @@
+    }
+
+    
+.card-description{
+    display : flex ;
+    flex-direction: column;
+    justify-content : space-around ;
+    font-size : 1.5rem ;
+    margin-left : 1rem ;
+}
+.card-title{
+    font-weight : bold;
+}
+.card-description-primary .card-primary-text{
+    color : var(--ca-primary-color) ;
+    font-style: italic;
+}
+.card-description-primary .card-price{
+    color : var(--ca-secondary-color);
+}
+.star-badge .far{
+    color : var(--danger-text) ;
+    cursor: pointer;
+
+}
+.star-badge .far:hover{
+    font-size : 1.8rem ;
+    cursor: pointer;
+
+}
+.product-description{
+    font-size : 1.5rem ;
+}
+.card-cta{
+    display : flex ;
+    flex-wrap : wrap ;
+    justify-content: flex-start;
+    align-items : center ;
+    gap : 2rem ;
+}
+.fa-heart{
+    color : var(--danger-text) ;
+    font-size : 2.5rem ;
+    font-weight: 500;
+    cursor: pointer;
+}
+.fa-heart:hover{
+    font-weight : 600;
+}
+.btn-cta{
+    color : white ;
+    background-color: var(--ca-primary-color);
+    border-radius: 0.5rem ;
+    padding : 1.5rem ;
+    border : none ;
+    cursor: pointer;
+}
+.btn-cta:hover{
+    background-color : var(--ca-primary-color-lighter)
+}
+/* card : horizontal and vertical */
+
+.card-horizontal {
+    max-width : 60% ;
+    display: flex;
+    flex-direction: row;
+    justify-content: center ;
+    align-content: center;
+    width: 100%;
+    gap: 1.2rem;
+}
+.card-image-horizontal {
+    width: 50rem;
+    height: 28rem ;
+    border: 0;
+    padding: 1rem;
+    position: relative;
+}
+.description-wrap{
+    display : flex ;
+    flex-direction: row;
+    justify-content: space-between;
+}
+.card-dismiss{
+    margin-right : 20px;
+    font-weight: bolder;
+    font-size : 3rem;
+    cursor:pointer;
+    
+}
+.card-dismiss:hover{
+    color : var(--ca-primary-color)
+}
+.card-vertical {
+    max-width : 30% ;
+    height : auto;
+    display: flex;
+    flex-direction: column;
+    justify-content: center ;
+    align-content: center;
+    gap: 0.8rem;
+    padding : 1rem ;
+}
+.card-image-vertical{
+    width: 100%;
+    height: auto ;
+    border: 0;
+    padding: 1rem;
+    position: relative;
+}
+.card-vertical .card-description{
+    gap : 2.0rem ;
+    padding-top : 1rem ;
+}
+.card-badge{
+    position : absolute ;
+    top : 0 ;
+    left : 0 ;
+    font-size : 1.5rem ;
+    background-color : var(--success-text);
+    color : var(--white-color);
+    padding : 1rem ;
+    border-radius : 3rem ;
+}
+
+/* Card : text overlay and text only */
+
+.card-text-overlay{
+    width : 45% ;
+}
+.card-text-overlay img{
+    position : relative ;
+    opacity : 75% ;
+}
+.overlay-text{
+    position : absolute ;
+    color : white ; 
+    font-size : 2rem ;
+    top : 75px;
+    text-decoration: underline solid var(--white-color);
+    text-underline-position: under;
+    cursor : pointer ; 
+}
+.card-text-only{
+    max-width : 45% ;
+    font-size: 2.0rem ;
+    height : 20vh ;
+    display: flex;
```

Comment:
![image](https://user-images.githubusercontent.com/65125998/153436296-51e0094c-ed42-47be-b1da-7840bd8a66d9.png)
Because you have given 
height: 20vh;
This is happening